### ISSUE TITLE
[FEATURE REQUEST] Use different strings for 1-month and 1-year full chem benchmark type in run scripts

### ISSUE THREAD

**Issue by lizziel** [geoschem/gcpy#87]
I noticed our 1-month and 1-year full chem benchmark scripts both have the same benchmark type variable. I think it would be more versatile if we distinguished between the two types.

For example, for budget we pass the benchmark type and it would be useful to determine if it's 1-month or 1-year from that variable in the logic to set units. Right now it checks if the seconds interval is greater than 3e7 to determine if it is 1-year.

Distinguishing between 1-month and 1-year would also enable logic to determine whether to use xarray open_dataset or open_mfdataset which merges multiple files.

**Comment by lizziel** [comment]
Quick update: I am looking through the latest GCPy and it appears we are still using the same benchmark type string for 1-month and 1-year benchmarks. I will leave this feature request open since distinguishing between 1-month and 1-year will allow greater flexibility.

**Comment by yantosca** [comment]
We could achieve the same thing by adding a duration flag to the YAML config files.  This might be cleaner.

**Comment by lizziel** [comment]
I agree. If duration is 1-year then it can go into the seasonal plots etc. If 1-month then it will just do a straight comparison.
